recovery/elliptics_recovery/recovery.py

```diff
@@ -55,6 +55,25 @@ def cleanup(path):
                 os.unlink(file_name)
 
 
+def get_routes(ctx):
+    log.debug('Requesting routes')
+
+    log.debug("Creating logger")
+    elog = elliptics.Logger(ctx.log_file, int(ctx.log_level))
+
+    log.debug("Creating node")
+    node = elliptics_create_node(address=ctx.address,
```

Comment:
All requests are sent from subprocesses and, currently, there is no possibility to pass initialized node from main process to subprocesses.